octoprintApis/client.go

```diff
@@ -140,7 +140,10 @@ func (this *Client) handleResponse(
 ) ([]byte, error) {
 	logger.TraceEnter("Client.handleResponse()")
 
-	defer httpResponse.Body.Close()
+	defer func() {
```

Comment:
`ImageFromUrl()`  (in utils/gtk.go) has:
```
defer response.Body.Close()
```

@freman in your opinion, should this also be changed to an IIFE?